CONTRIBUTING.md

```diff
@@ -2,11 +2,15 @@
 
 ## Development environment
 
+The following requires poetry 1.2.0a2 or newer.
+
 Create Python env
 
 ```shell
 make env
 source ./.venv/bin/activate
+conda create --name s3contents
```

Comment:
`make env` uses poetry now instead of conda so i dont think this is needed.

Reply:
I have unable to make it work, after `make env` I don't have `./.venv/bin/activate` file, which makes sourcing impossible

I separated the explanation to make it clear that this is not the preferred solution but still keep it for people who are in my case for now. Is that ok ?

Reply:
Ah got it. It makes sense because i have some global poetry config.
This is perfect, thank you!